cranelift/codegen/src/isa/aarch64/inst/regs.rs

```diff
@@ -179,6 +179,18 @@ pub fn pretty_print_reg(reg: Reg) -> String {
     show_reg(reg)
 }
 
+fn show_reg_sized(reg: Reg, size: OperandSize) -> String {
+    match reg.class() {
+        RegClass::Int => show_ireg_sized(reg, size),
+        RegClass::Float => todo!(),
```

Comment:
`todo!()` here -- can we just show the reg (and below for vector too) with `show_reg` for now?